geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/build

```diff
@@ -5,21 +5,31 @@ import os
 import os.path
 import platform
 import re
+import shlex
 import shutil
 import stat
 import subprocess
 import sys
 import urllib.request
-from typing import Any, List
+from typing import TYPE_CHECKING, Any, List, Optional
 
 import yaml
 
+CompletedProcess = subprocess.CompletedProcess[str] if TYPE_CHECKING else subprocess.CompletedProcess
 
-def run(args: argparse.Namespace, command: List[str], **kwargs: Any) -> None:
+
+def run(
+    args: argparse.Namespace, command: List[str], exit_on_error: bool = False, **kwargs: Any
+) -> Optional[CompletedProcess]:
     if args.verbose or args.dry_run:
-        print(" ".join(command))
-    if not args.dry_run:
-        subprocess.run(command, **kwargs)  # nosec
+        shlex.join(command)
+    if not args.dry_run or "stdout" in kwargs:
+        process = subprocess.run(command, **kwargs)  # nosec
+        if exit_on_error and process.returncode != 0:
+            print("Error on running {}".format(shlex.join(command)))
+            sys.exit(process.returncode)
```

Comment:
Does not CalledProcessError or stderr contain more info about what's going wrong.

I suspect this mask the error description.

Reply:
Example in separate example (executing `run(args, ['ls', '-l', 'toto'], exit_on_error=True)`):

Previous result:
```
ls: impossible d'accéder à 'toto': Aucun fichier ou dossier de ce type
Traceback (most recent call last):
  File "/home/sbrunner/workspace/c2cgeoportal/test.py", line 61, in <module>
    run(args, ['ls', '-l', 'toto'], exit_on_error=True, check=True)
  File "/home/sbrunner/workspace/c2cgeoportal/test.py", line 27, in run
    process = subprocess.run(command, **kwargs)  # nosec
  File "/usr/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['ls', '-l', 'toto']' returned non-zero exit status 2.
```

Current result:
```
ls: impossible d'accéder à 'toto': Aucun fichier ou dossier de ce type
Error on running ls -l toto
```

The exception hides a bit the real error for the end user, but it's true that the exception contains the line where the error comes from.
Should I add a new option `--trace` to add the exception?

Reply:
Seems ok, but "error on running" seems wreid to me. "Error while running" or "An error occured during execution of"

Reply:
Done :-)